src/core/os/amdgpu/amdgpuDevice.cpp

```diff
@@ -1986,6 +2005,24 @@ Result Device::CreateInternalImage(
     return result;
 }
 
+// =====================================================================================================================
+// Returns true if gpu + amdgpu kms driver do support 16 bit floating point display.
+bool Device::HasFP16DisplaySupport(void)
+{
+    bool supported = false;
+
+    // On Linux 5.8 (DRM 3.38) and later we also have the 64 bpp fp16 floating point format
+    // on display engines of generation DCE 11.2 - DCE 12, and all DCN engines, iow. Polaris
+    // and later.
+    if ((IsDrmVersionOrGreater(3, 38) || IsKernelVersionEqualOrGreater(5, 8)) &&
+        (IsGfx10Plus(m_chipProperties.gfxLevel) || IsGfx9(*this) ||
+        (IsGfx8(*this) && (IsPolaris10(*this) || IsPolaris11(*this) || IsPolaris12(*this))))) {
```

Comment:
"{" should occupy one new line with proper alignment.

Reply:
Thanks. Done, in just updated pull request.